tsconfig.json

```diff
@@ -23,5 +23,9 @@
     "moduleResolution": "node",
     "target": "ES2020",
     "strict": true,
+    "outDir": "build",
+
+    // This does not work understand enums so we cannot leave it enabled.
+    // "importsNotUsedAsValues": "error"
```

Comment:
Great to document, but documentation is not great: "does not work understand"??

Also: is it just getting upset about how we reexport enums from `namespaces`, or is this feature somehow fundamentally broken in some way that the TypeScript maintainers should be appraised of?

Reply:
Fixed, it is not a problem with `namespaces`. I believe Christopher is digging into the actual issue, and filing a bug.

Reply:
Added a comment linking to https://github.com/microsoft/TypeScript/issues/49974